frontend/src/app/password-reset/password-reset.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, ViewChild } from '@angular/core';
+import { NgForm } from '@angular/forms';
+
+import { AuthService } from '../auth.service';
+
+
+@Component({
+  selector: 'volontulo-password-reset',
+  templateUrl: './password-reset.component.html',
+  styleUrls: ['./password-reset.component.scss'],
+})
+export class PasswordResetComponent {
+  @ViewChild('resetForm') resetForm: NgForm;
+  alertSuccessClosed = true;
+
+  constructor(private authService: AuthService) { }
+
+  onSubmit(): void {
+    this.authService.resetPassword(this.resetForm.value)
```

Comment:
Unless you plan to extend the method in a service, use `this.resetForm.value.username` as a parameter to pass a string, not the object.

Reply:
:+1: